src/extras/Flowmap.js

```diff
@@ -105,6 +105,16 @@ export class Flowmap {
         });
         this.mask.swap();
     }
+
+    dispose() {
+        this.mask.read.dispose();
+        this.mask.write.dispose();
+        delete this.mask;
+
+        this.mesh.program.dispose();
+        this.mesh.geometry.dispose();
+        delete this.mesh;
```

Comment:
Should Mesh have its own disposal method where it will dispose of its program and geometry? My concern would be with shared programs/geometry and whether it's a good idea to let the user choose -- and thus open up to renderer or transform-level disposal.

Reply:
Thanks again for your work!
This is a really good question - I share your concern for shared elements, however I suppose that's just something the developer has to control. Perhaps if there's a way to flag the geometries/programs to ignore disposal?

Reply:
I know that three requires people to manually dispose of everything (and know some things like bones are textures and thus need to be disposed of). This is a bit of a chore, so I was curious if we could do better or keep things consistent.

Something I've seen with three (& react-three-fiber, specifically) is people overwriting the dispose property to null or similar, and that would preserve its properties and descendants. The caveat is that this has to be done on a class like Transform to preserve methods for manual disposal, so I'm not sure if this is useful for things outside of the scene graph like shared uniforms, etc. I think we would indeed need a property specific to configure that behavior, if at all.

Reply:
I think I'll only be eager to dispose of things internal to a class. Leaving the rest to the user can be annoying, but it only enables advanced usage.